sb_vision/tokens.py

```diff
@@ -130,23 +132,36 @@ def _apply_distance_model_component_to_homography_matrix(
     )
 
 
+# TODO: Don't hard-code this
```

Comment:
This needs resolving before we can merge.

I suggesting adding a field to the calibrations which records the size.